Bump dependency react-icons from v2 to v3

emulienfou opened the pull request:
Bumping use of lib `react-icons` v2 to v3

lijinke666 commented:
Thanks.

like reload icon size is different and delete and close icon no alignment
![image](https://user-images.githubusercontent.com/21015895/91629573-b96a4000-e9fc-11ea-9e07-6cf78f2372cf.png)

emulienfou commented:
@lijinke666 I updated the reload icon displaying the same has previously. I also align `delete` and `close` icons using flex, and I also fixed the line separator color in light theme.

lijinke666 commented:
/rebase
